shell/platform/tizen/channels/encodable_value_getter.h

```diff
@@ -0,0 +1,31 @@
+// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/common/client_wrapper/include/flutter/encodable_value.h"
+#include "flutter/shell/platform/tizen/logger.h"
```

Comment:
I guess this got included while debugging.

Reply:
Thanks !